app/src/renderer/components/common/TmSessionHardware.vue

```diff
@@ -44,16 +51,21 @@ export default {
     setStatus(value) {
       this.status = value
     },
+    setConnectionError(error) {
+      this.connectionError = error
+    },
     async connectLedger() {
+      this.setConnectionError(null)
       this.setStatus(`detect`)
-      const connected = await this.$store.dispatch(`connectLedgerApp`)
-      if (connected) {
+      const error = await this.$store.dispatch(`connectLedgerApp`)
```

Comment:
this should throw instead of returning an error imo

Reply:
updated